_source/logzio_collections/_security-sources/axonius.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Ship logs from Axonius
+logo:
+  logofile: axonius.png
+  orientation: vertical
+data-source: Axonius
+templates: ["network-device-filebeat"]
+contributors:
+  - nshishkin
+shipping-tags:
+  - firewalls
+order: 1380
+---
+[Axonius](https://www.axonius.com/) is a cybersecurity asset management platform. This topic describes how to send system logs from your Axonius platform to Logz.io. 
+
+**Before you begin, you'll need**:
+
+* an active account with Axonius
+* an active account with Logz.io
+* Filebeat 7 installed on your machine
+* Root priveleges on your machines 
+
+<div class="tasklist">
+
+
+##### Configure Axonius to send syslog notifications to a remote Syslog server running Filebeat
+
+1. On your Axonius web interface, go to **System Settings > Global Settings > Syslog Settings**.
+![Axonius](https://dytvr9ot2sszz.cloudfront.net/logz-docs/axonius/axonius.png)
+2. Select **Use Syslog**.
+3. Enter the IP address of your remote syslog server into the **Syslog host** field.
```

Comment:
If i understand correct the filebeat is acting as a syslog server? Can you write few words on it in the doc?
Something like
Enter the IP address of your remote syslog server (where filebeat is installed) into the **Syslog host** field.